.github/workflows/release.yml

```diff
@@ -101,6 +108,7 @@ jobs:
           mv ./target/release/sui-indexer ./target/release/sui-indexer-${os_type}
           mv ./external-crates/move/target/release/move-analyzer ./external-crates/move/target/release/move-analyzer-${os_type}
           tar -cvzf ./target/release/sui-${{ env.sui_tag }}-${os_type}.tgz ./target/release/sui*-${os_type}* ./external-crates/move/target/release/move-analyzer-${os_type}
+          aws s3 cp ./target/release/sui-${{ env.sui_tag }}-${os_type}.tgz s3://sui-releases/releases/sui-${{ env.sui_tag }}-${os_type}.tgz
```

Comment:
We can probably get away with only pushing these on `mainnet` release

Reply:
Added the check